lib/wda-mac.js

```diff
@@ -260,6 +261,9 @@ class WDAMacServer {
     this.process = new WDAMacProcess();
```

Comment:
it would still make sense to assign `process` to `null` in the constructor by default